.travis.yml

```diff
@@ -16,8 +19,10 @@ install:
   - if [[ $GROUP == 'js' ]]; then pip install 'tornado<4'; fi
   - cd ipython && git submodule update && pip install -e ".[notebook]" && cd ..
   - pip install -r requirements.txt
-  - pip install nose requests!=2.4.2
+  - pip install nose requests!=2.4.2 fabric
   - python setup.py install
   - if [[ $GROUP == 'js' ]]; then cp nbextensions/* $(ipython locate)/nbextensions/; fi
 script:
   - ./scripts/runtests $GROUP
+after_success:
+  - if [[ $GROUP != 'js' ]]; then fab travis_docs; fi
```

Comment:
Yeah, just updated it to only do not-PRs and I will update it to only do it on master just as soon as I have it working :smile: